package.json

```
@@ -18,11 +19,13 @@
     "test:cypress:run": "cypress run",
     "test:unit": "kcd-scripts test --no-watch",
     "test:unit:watch": "kcd-scripts test",
+    "typecheck": "dtslint ./types/",
     "validate": "kcd-scripts validate build,lint,test"
```

Let's add typecheck to validation

```suggestion
    "validate": "kcd-scripts validate build,lint,test,typecheck"
```